### PR TITLE
MC-11021: Added delete public ip

### PR DESCRIPTION
**Pull request by franzpgarcia**

### CODE REVIEW

File: source/includes/azure/_public_ip_addresses.md [hunk 1]
```diff
@@ -98,3 +98,18 @@ Attributes | &nbsp;
...
 `ipVersion` <br/>*string* |  IP version of the public IP address. Possible values: `IPV4`, `IPV6`.
+
+
+<!-------------------- DELETE A PUBLIC IP -------------------->
+
+#### Delete a public IP address
+
+```shell 
+curl --request DELETE \
+  --url http://cloudmc_endpoint/v1/services/azure/example/publicipaddresses/subscriptions/6b6a1f27-55c1-4b1d-969b-60a3c9eebe64/resourceGroups/cmc-example/providers/Microsoft.Network/publicIPAddresses/some-public-ip \
```
Comment (fyrnaga) [inline source/includes/azure/_public_ip_addresses.md]: offuscate the subscription id

Comment (franzpgarcia) [reply source/includes/azure/_public_ip_addresses.md]: done thanks.

**Review by fyrnaga — APPROVED** [review]
Resolve the conflicts and merge

**Review by GuillaumeVialle — APPROVED** [review]
LGTM, you have a conflict to resolve

**Pull request merged**